rep-0008.rst

```diff
@@ -67,12 +133,12 @@ Module with msgs/srvs::
           |- yourfiles.py
      |- scripts/
         |- non-exported python files
-    
+
 We distinguish between "nodes" and "scripts" for clarity to
 users. Nodes are executable Python files that conform to the ROS node
 API. Scripts are executable Python files that do not conform to the
 ROS node API.
```

Comment:
Yeah, we were discussing the need to expand the catkin section with respect to scripts above. I'll say something similar in the catkin section.